src/styles/blocks/star.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+}
+
+.stars--rating-4 .stars__star:nth-child(-n + 4) {
+  background-image: url('../images/star-active.svg');
```

Comment:
Selectors must be written in kebab-case. Please rename the selector to follow the convention, for example '.stars--rating-4 .stars__star' should be '.stars--rating-4 .stars__star'.